utils/artemis_utils/__init__.py

```diff
@@ -278,6 +283,16 @@ def flatten(items, seqtypes=(list, tuple)):
     return res
 
 
+def load_json(filename):
```

Comment:
added missing functions from the original monitor code utils (that file is deleted in the current PR).